diagnostics/stratonovich_general.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import torch
+
+from tests.problems import Ex4
+from torchsde import BrownianInterval
+from torchsde.settings import LEVY_AREA_APPROXIMATIONS, SDE_TYPES
+from . import inspection
+from . import utils
+
+
+def main():
+    small_batch_size, large_batch_size, d, m = 16, 8192, 3, 5
+    t0, t1, steps, dt = 0., 2., 10, 1e-1
+    ts = torch.linspace(t0, t1, steps=steps, device=device)
+    dts = tuple(2 ** -i for i in range(1, 8))  # For checking strong order.
+    sde = Ex4(d=d, m=m, sde_type=SDE_TYPES.stratonovich).to(device)
+    methods = ('midpoint', 'log_ode')
+    img_dir = os.path.join('.', 'diagnostics', 'plots', 'stratonovich_general')
```

Comment:
This depends upon the run location. Can this be switched to e.g:
```
import pathlib
here = pathlib.Path(__file__).resolve().parent
img_dir = here / 'plots/ ...
```

Reply:
Good point. Fixed with 
```
img_dir = os.path.join(os.path.dirname(__file__), 'plots', ...)
```